packages/components/src/dropdown/test/index.tsx

```diff
@@ -24,14 +24,13 @@ describe( 'Dropdown', () => {
 					</button>
 				) }
 				renderContent={ () => <span>test</span> }
-				popoverProps={ { 'data-testid': 'popover' } }
```

Comment:
This caused a TS error:
<img width="958" alt="Screen Shot 2022-12-07 at 8 36 11 PM" src="https://user-images.githubusercontent.com/4063887/206342581-d052a2e2-7392-4124-84a1-aa5e0e241c49.png">

Reply:
Oh, good to see we're not the only ones to see that